calculator.c

```diff
@@ -0,0 +1,77 @@
+#include<stdio.h>
+#include<stdlib.h>
+#include<ctype.h>
+#include<string.h>
+void removewhitespacesfromexpression(char input[]) {

```

Comment:
apply code formatting and match function names with this standards: 

- Use snake_case for function names.
- Start function names with a verb to indicate action.
- Keep names concise but meaningful.

ex : 
int calculate_sum(int a, int b);
void display_message(const char *message);